notebooks/en/enterprise_cookbook_dev_spaces.md

```diff
@@ -0,0 +1,88 @@
+
+In the bottom left of the image, you can also see the `Sleep time settings` where you can define how long you want the hardware to run in case of inactivity. This is a major advantage over Google Colab: if you want to save money, you can make the Space sleep after 15 minutes of inactivity, but if you need the hardware to be available for a 48 hour training run or longer, you can just prevent the Space from falling asleep and let it run for as long as you want. You can also manually `Pause` the Space and you will no longer be charged for the Space Hardware. 
+
+<img src="https://cdn-lfs-us-1.huggingface.co/repos/13/3d/133d8ca2460bf82ba2bdbe928d91a6c780364a6d0cf9005087db081cca492c02/ce651a2be35a88878755f41531c877b50ba194fce0971397010e2271904bbf39?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27enterprise-jupyterlab-hardware-options.png%3B+filename%3D%22enterprise-jupyterlab-hardware-options.png%22%3B&response-content-type=image%2Fpng&Expires=1717920105&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcxNzkyMDEwNX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy11cy0xLmh1Z2dpbmdmYWNlLmNvL3JlcG9zLzEzLzNkLzEzM2Q4Y2EyNDYwYmY4MmJhMmJkYmU5MjhkOTFhNmM3ODAzNjRhNmQwY2Y5MDA1MDg3ZGIwODFjY2E0OTJjMDIvY2U2NTFhMmJlMzVhODg4Nzg3NTVmNDE1MzFjODc3YjUwYmExOTRmY2UwOTcxMzk3MDEwZTIyNzE5MDRiYmYzOT9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoifV19&Signature=Ye2vpEAbFaQuVTru4iLEQncnF4VzON9mAEmkJqjyJQy9J0S4kA%7EtO3O3LwWFVHOT2esYGDjG1re0lwYLG3mft1lxieoJbm5vuwIPNELMSa-Vb3e7bA6ZuOcisklEoTwpRi0aQcKa6NlWoF3rNDFLwmLdZVQNHvFKCee6%7E0wiRol-znACCk-7uIoJ7dclyRmVmUgdsgskVfSVIXDYVZH%7EQyA5yH4AhNIwQhb5gNjhVZq0OG%7EbmCGkdszhLVD5ljw4E6Lage5kr1ID8owZFNI62jKbNHvIJS-MM0WtN3iu0zCt9mv28Py0A-UKDrBmbSvJQuCwvcRAlhjiLUPyApct2g__&Key-Pair-Id=KCD77M1F0VK2B">
+
+If you scroll down in the settings, you will see additional options, like expanding storage, resetting the space etc. In case you have not set a password during Space creation, you can also create a secret called `JUPYTER_TOKEN` here later, which will replace the default "huggingface" password.
+
+> [!TIP]
+> When you actively work with the space over several days or weeks, files will start accumulating in the storage cache. When you get a warning that the storage is full and you think that the storage quote should not be reached yet, it might be helpful to factory reset the Space to empty the cache. 
```

Comment:
```suggestion
> When you actively work with the Space over several days or weeks, files accumulate in the storage cache. When you get a warning that the storage is full and you think that the storage quote should not be reached yet, it might be helpful to factory reset the Space to empty the cache. 
```

Reply:
Maybe we can mention attaching persistent storage?https://huggingface.co/docs/hub/en/spaces-storage#persistent-storage

Reply:
I had this issue of data accumulating in the cache with persistent storage as well and persistent storage is mentioned further above. Implemented the other suggestions for the paragraph